pkg/kapp/cmd/app/logs_flags.go

```diff
@@ -8,17 +8,19 @@ import (
 )
 
 type LogsFlags struct {
-	Follow       bool
-	Lines        int64
-	ContainerTag bool
-	PodName      string
+	Follow         bool
+	Lines          int64
+	ContainerNames []string
+	ContainerTag   bool
+	PodName        string
 }
 
 func (s *LogsFlags) Set(cmd *cobra.Command) {
 	cmd.Flags().BoolVarP(&s.Follow, "follow", "f", false, "As new pods are added, new pod logs will be printed")
 	cmd.Flags().Int64Var(&s.Lines, "lines", 10, "Limit to number of lines (use -1 to remove limit)")
 	cmd.Flags().BoolVar(&s.ContainerTag, "container-tag", true, "Include container tag")
 	cmd.Flags().StringVarP(&s.PodName, "pod-name", "m", "", "Set pod name to filter logs (% acts as wildcard, e.g. 'app%')")
+	cmd.Flags().StringSliceVarP(&s.ContainerNames, "container-names", "c", []string{}, "Set container names to filter logs in the pod, separated by comma")
```

Comment:
actual flag name should be singular "container-name" to be more consistent with rest of kapp flags. default could be nil, instead of []string{} (since it's a typed field)